models/User.go

```diff
@@ -740,7 +740,19 @@ func PlaceAskOrder(userId uint32, ask *Ask) (uint32, error) {
 		return 0, NotEnoughStocksError{currentAllowedQty}
 	}
 
-	l.Debugf("Check2: Passed. Creating Ask.")
+	l.Debugf("Check2: Passed.")
+
+	var orderFee = uint32((ORDER_FEE_PERCENT / 100.0) * float32(ask.StockQuantity*ask.Price))
+	var cashLeft = uint32(user.Cash) - orderFee
```

Comment:
```
orderFee := uint32((ORDER_FEE_PERCENT / 100.0) * float32(ask.StockQuantity*ask.Price))
cashLeft := uint32(user.Cash) - orderFee
```
for keeping with golang conventions (maintain this in the rest of your changes as well)